plugin/description.html

```diff
@@ -1 +1,25 @@
-The plugin provides support for Rust language
+The plugin adds support for Rust language. Enjoy the full IDE experience featuring:
+<ul>
+    <li>
+        Smart coding assistance
+    </li>
+    <li>
+        Seamless Cargo support
+    </li>
+    <li>
+        Easy access to remote environments
```

Comment:
Is it about run targets?

Reply:
I see now that it could be clearer. I changed it to "Remote targets". What do you think?